src/app/components/msp/application/prepare/prepare.component.html

```diff
@@ -6,7 +6,8 @@
 
 <common-consent-modal #mspConsentModal [isUnderMaintenance]="true" (onClose)="liveInBCBtn.focus()"
         [title]="'Information collection notice'" agreeLabel="I have read and understand this information"
-        [processName]="lang('./en/index.js').consentProcessName" (accept)="mspApplication.infoCollectionAgreement = $event;  this.dataService.saveMspApplication();">
+        [processName]="lang('./en/index.js').consentProcessName" 
```

Comment:
This  is still using an i18n / en folder.  Those folders should be deleted entirely and the value of "consentProcessName" moved into the top component.